sound/CC-Sounds/mon_death/zombie_gibbed/zombie_gibbed.json

```diff
@@ -1,12 +1,11 @@
 [
-    {
-        "type": "sound_effect",
-        "id" : "mon_gibbed",
-        "variant" : "zombie_gibbed",
-        "volume" : 100,
-        "files" : [
-            "mon_gibbed/zombie_gibbed_1.ogg",
-            "mon_gibbed/zombie_gibbed_2.ogg"
-        ]
-    }
-]
\ No newline at end of file
+  {
+    "type": "sound_effect",
+    "id": "mon_gibbed",
+    "variant": "zombie_gibbed",
+    "volume": 100,
+    "//file1": "TODO: mon_gibbed/zombie_gibbed/zombie_gibbed_1.ogg",
```

Comment:
Why the todo?

Reply:
It looks like those sound files don't exist yet, so I moved them to comments instead (The workflow will fail if it detects missing files in the `files` field).